docs/guides/qiskit-transpiler-service.mdx

```diff
@@ -96,9 +96,9 @@ The following examples demonstrate how to transpile circuits using the Qiskit Tr
 
```

Comment:
Fix grammatical error: 'Following is' should be 'Following are' since it refers to multiple limitations